DMTN-190.tex

```diff
@@ -133,16 +268,72 @@ \section{Estimating uncertainties}
 \end{equation}
 The proof follows by considering $\int\rmd{\bf x} f({\bf x}) = \tilde{f}(0)$, where $\tilde{f}$ is the Fourier transform of $f$.
 
-\subsection{Special case of Gaussian kernels}
-Suppose $I({\bf x})$ has a circular Gaussian PSF of size $s$ and the target PSF is a Gaussian PSF of size $p = fs$ ($f>1$), then the kernel $K$ is also a Gaussian of size $s(f^2-1)^{1/2}$. 
+\section{Concluding remarks}
+This technote presents the Gaussian Aperture and PSF photometry algorithm and covers various technical details with regards to its implementation in the Rubin Science Pipelines.
+This note also captures various calculations that are used in testing the code, since they are cumbersome to leave as comments within the code itself.
+
+\appendix
+% Include all the relevant bib files.
+% https://lsst-texmf.lsst.io/lsstdoc.html#bibliographies
+\section{References} \label{sec:bib}
+\renewcommand{\refname}{} % Suppress default Bibliography section
+\bibliography{local,lsst,lsst-dm,refs_ads,refs,books}
+
+% Make sure lsst-texmf/bin/generateAcronyms.py is in your path
+\section{Acronyms} \label{sec:acronyms}
+\input{acronyms.tex}
+% If you want glossary uncomment below -- comment out the two lines above
+%\printglossaries
+
+\begin{comment}
```

Comment:
This comment looks complete but also doesn't show up in the PDF as far as I can tell, so let me know what you want done with it, if anything.

Reply:
Nothing. Let it be stashed for now.